src/components/MultiSelect.js

```diff
@@ -49,8 +55,35 @@ const MultiSelectItem = ({
   multiSelectedItemContainerStyle,
   multiSelectedItemLabelStyle,
   disabled,
+  confirmationAlertObj = {},
 }) => {
   const theme = useContext(ThemeContext);
+  const {
+    alertTitle = "Delete tag",
+    alertDescription = `Are you sure you want to delete tag: ${label}? This action cannot be undone.`,
+    alertConfirmButtonLabel = "Delete",
```

Comment:
@Tsudhishnair Instead of `Delete` we can use `Proceed` right? We can pass it as a prop. But since we follow uniformity across the projects it's better to have the same label in the library.